docs/configuration/authentication/jwt.md

```diff
@@ -11,12 +11,16 @@ SECRET = "SECRET"
 
 auth_backends = []
 
-jwt_authentication = JWTAuthentication(secret=SECRET, lifetime_seconds=3600)
+jwt_authentication = JWTAuthentication(secret=SECRET, lifetime_seconds=3600, tokenUrl="auth/jwt/login")
```

Comment:
Shouldn't the default parameter here be `tokenUrl="login"`, in accordance with the class definition?

Reply:
I think it's okay to have `auth/jwt/login` because it's the working value when configuring the JWT authentication router when following the documentation. If the user follows this, it should have a working example out-of-the-box (like full examples).

Reply:
I think this can make sense, since you've specified that the `tokenUrl` "defaults to `login`".

Reply:
Maybe I should change the default actually 🤔